js/PickerWindows.windows.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ * @flow strict-local
+ */
+
+'use strict';
+
+import React from 'react';
+import {
+  processColor,
+  requireNativeComponent,
+  NativeModules,
+  StyleSheet,
+} from 'react-native';
+
+const REF_PICKER = 'picker';
+
+import type {SyntheticEvent} from 'CoreEventTypes';
+import type {TextStyleProp} from 'StyleSheet';
+
+const RCTPicker = requireNativeComponent('RCTPicker');
```

Comment:
@kaiguo this will pull in the built in view manager as well, and will cause crashes in 0.62 if we include this due to duplicate registration of the same component.

Reply:
Yeah we should remove the built in version in 62. Or are you suggesting using a different module name here?

Reply:
It's not removed from upstream yet, only deprecated, so we shouldn't remove ours. We will want to rename this one. I would follow whatever the other platforms are doing for name here.

We also still need to support this on 0.61, and wouldn't want the two to point to the same view manager.

Reply:
Makes sense, I'll rename it.